setup.cfg

```diff
@@ -0,0 +1,67 @@
+[metadata]
```

Comment:
Cool! This is the new format @matt-hayden had mentioned, right?

Reply:
Yep! All of this is pretty much identical to what was in `setup.py`,